hwdb/20-vmbus-class.hwdb

```diff
@@ -0,0 +1,61 @@
+# This file is part of systemd.
```

Comment:
This file needs to be added to `hwdb/meson.build`.

Reply:
meson.build is not on the main branch

Reply:
It is in latest